packages/trader/src/RangeTrader.ts

```diff
@@ -0,0 +1,177 @@
+import winston from "winston";
+import Web3 from "web3";
+import BigNumber from "bignumber.js";
+const ExchangeAdapterInterface = require("./exchange-adapters/ExchangeAdapterInterface");
+
+const { ConvertDecimals, createFormatFunction, createObjectFromDefaultProps } = require("@uma/common");
+import assert from "assert";
+
+export class RangeTrader {
+  readonly normalizePriceFeedDecimals: any;
+  readonly formatDecimalString: any;
+
+  readonly BN: any;
+  readonly toBN: any;
+  readonly toWei: any;
+  readonly fromWei: any;
```

Comment:
I really like this pattern but could not get it to work :/ I was however able to do this fine which is still measurably cleaner:
```
import Web3 from "web3";
const { toWei, toBN } = Web3.utils;
```

I also added a new function:
```
const toBNWei = (number: string | number) => toBN(toWei(number.toString()).toString());
```
to clean up some of the conversions through. this is worse than normal JS as typescript gets upset with things like
```
toBN(toWei(numberThatIsAString))
```

as apparently `toWei` can return a Number so you end up needing to do horrible things like:
```
toBN(toWei(yourNumber.toString()).toString()
```

Reply:
haha, gross.

I actually think this is the really nice thing about typescript. We've had so many difficult-to-diagnose errors because of the input/output types of the web3 functions. I think ts will finally force us to handle these correctly rather than waiting for runtime errors. It's a little messier, but it's more fool-proof :)